### ISSUE TITLE
can not run castero

### ISSUE THREAD

**Issue by Monirzadeh** [xgi/castero#59]
hi 
i install castero with this command
`pip3 install castero`
but
`castero
zsh: command not found: castero
`

i use zsh and kbuntu 19.10

**Comment by xgi** [comment]
Can you try running `python -m castero`? If that works, it means that your PATH does not include the one that the castero binary got installed to. I think that location can depend on your system configuration, but maybe check if you can find the castero executable in `/usr/bin` or `~/.local/bin` and add that directory to your PATH if so.

**Comment by Monirzadeh** [comment]
`python -m castero` not working
but castero is in '~/.local/bin' directory
 i don't get it

> add that directory to your PATH if so

how should i do that.
why castero go in `/usr/bin` ?

**Comment by rien333** [comment]
> why castero go in /usr/bin
@Monirzadeh most packages/software goes there, at least by default. 

You can check the current contents of your PATH variable with: 
```
echo $PATH
```

To permanently add something to your path in zsh, you can follow [this answer](https://stackoverflow.com/a/50233328/1657933)  (or any of the other answers on the page if you don't like this method). Be sure to check if castero is in ~/.local/bin/ or  /usr/bin first though. (I do find it unlikely that `/usr/bin/` wouldn't already be in your path, so check that as well)

**Comment by Monirzadeh** [comment]
> > why castero go in /usr/bin
> 
> @Monirzadeh most packages/software goes there, at least by default.
> 
> You can check the current contents of your PATH variable with:
> 
> ```
> echo $PATH
> ```
> 
> To permanently add something to your path in zsh, you can follow [this answer](https://stackoverflow.com/a/50233328/1657933) (or any of the other answers on the page if you don't like this method). Be sure to check if castero is in ~/.local/bin/ or /usr/bin first though. (I do find it unlikely that `/usr/bin/` wouldn't already be in your path, so check that as well)

thanks
i can do that but i don't like more and more option to my zshrc (that slow down zsh startup by the time)
 so can is there any way to run castero from /usr/bin ,etc... like other program 
it is my list of currently PATH variable value
`/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games`

**Comment by xgi** [comment]
I'm assuming this was caused by a user-level config setting, for Python or pip. I found [this thread](https://unix.stackexchange.com/questions/240037/why-did-pip-install-a-package-into-local-bin) although it doesn't give much useful information. I'm closing this since it doesn't seem to be something I can fix in castero, but if someone else has this issue and thinks there's an installation setting I can change, let me know.
